components/ui/Dialog.tsx

```diff
@@ -60,7 +60,7 @@ const DialogHeader = ({ className, ...props }: React.HTMLAttributes<HTMLDivEleme
 DialogHeader.displayName = 'DialogHeader';
 
 const DialogFooter = ({ className, ...props }: React.HTMLAttributes<HTMLDivElement>) => (
-  <div className={cn('flex flex-col-reverse sm:flex-row sm:justify-end sm:space-x-2', className)} {...props} />
+  <div className={cn('flex flex-col-reverse sm:flex-row sm:space-x-2', className)} {...props} />
```

Comment:
@gustavlrsn I removed `sm:justify-end` because it was preventing me from properly positioning multiple buttons on the interface. Not sure if we should keep alignment in such a generic component.

Reply:
I think we should keep `sm:justify-end` since it seems like a reasonable default to place the buttons at the end:

<img width="610" alt="image" src="https://github.com/opencollective/opencollective-frontend/assets/1552194/8b1b41f4-4037-499d-acfb-594bef3f5122">

It should work as you already implemented to modify this (because of the `cn` helper that uses tailwind-merge) through the className property:
```
        <DialogFooter className="sm:justify-between">
``